test/e2e/util.go

```diff
@@ -1918,12 +1937,28 @@ func isNodeReadySetAsExpected(node *api.Node, wantReady bool) bool {
 	return false
 }
 
+func isNodeOutOfDiskSetAsExpected(node *api.Node, wantOutOfDisk bool) bool {
```

Comment:
Done.

Sorry about that. I was afraid to refactor more code, i.e. code that depended on existing isNodeReadySetAsExpected().